installer/release/definitions.mk

```diff
@@ -0,0 +1,5 @@
+version := 1.3.4-SNAPSHOT
+cloudflow_installer_image_tag := 1.3.4-SNAPSHOT
+cloudflow_operator_image_tag := 1.3.4-SNAPSHOT
+
+cloudflow_spark_operator_image := 1.3.1-OpenJDK-2.4.5-1.1.0-cloudflow-2.12
```

Comment:
Likely this must be updated.

Reply:
Between 1.3.1 and 1.3.3, we didn't do anything to the Spark operator image and therefore didn't update its tag to match the 1.3.3 version. I can push a 1.3.3 tag to docker hub.

Reply:
ok good.